### ISSUE TITLE
"Device already configured" when I've used a phone number

### ISSUE THREAD

**Issue by myleshorton** [getlantern/lantern#4341]
Not sure what the best ultimate behavior is here, but the current setup definitely makes testing difficult. It might be better to just send another SMS for now at least? Otherwise how to I reset the data for a given phone number?

**Comment by percyalpha** [comment]
We need to let people use a phone number more than once. Suppose I want to set up desktop at home, work and on android

**Comment by myleshorton** [comment]
Yeah there are different calls for authorizing extra devices, so I don't believe it conflicts with that, but agreed we should make sure @percyalpha

**Comment by uaalto** [comment]
@myleshorton it replies with that error if the same device is used, but not if you want to change it. What about removing that check *only in staging*?

**Comment by uaalto** [comment]
This solution is getting deployed ATM. If we want the same behaviour in production we can add it.

**Comment by uaalto** [comment]
should we close this?

**Comment by qinmeilin** [comment]
please close it
